frontend/src/components/index.ts

```diff
@@ -0,0 +1,6 @@
+export { default as Navbar } from "./Navbar";
+export { default as Loader } from "./Loader";
+
+export { default as RenderEmbeddings } from "./RenderEmbeddings";
+export { default as EmbeddingDetails } from "./EmbeddingDetails";
+export { default as Footer } from "./Footer";
```

Comment:
I don't think you need index files. But LMK if I'm missing something.

Reply:
Placing the index inside a folder helps to improve my code organization, simplifies imports and provides a entry point for the folder's functionality. However, I'm open to using alternative approaches as well.